### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by danielpza**
- added a typescript check.
- fixed some typescript warnings.

### CODE REVIEW

File: src/index/getFilePaths.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ const globSearch = (pattern: string) => {
 };
 
 /** Recursively get all file paths. */
-export const getFilePaths = (paths: string[]) => {
+export const getFilePaths = (paths: string[], _options: {}) => {
```
Comment (sQVe) [inline src/index/getFilePaths.ts]: Instead of adding `_options` I think we should remove passing in options to the `getFilePaths` function.

Comment (danielpza) [reply src/index/getFilePaths.ts]: done, I thought perhaps it was going to be used later

Comment (sQVe) [reply src/index/getFilePaths.ts]: 👌

Could you also remove `noEmit` from the check script? It is already set in this tsconfig.

**Comment by sQVe** [comment]
@danielpza Awesome. Thank you for your contribution! :tada:

**Pull request merged**